CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Removed OS-specific configuration directory.
   - Linux: $HOME/.config/giganto/config.toml
   - macOS: $HOME/Library/Application Support/com.cluml.giganto/config.toml
+- Removed the GraphQL API `tsvFormattedRawEvents`.
```

Comment:
I think this line should be 
```suggestion
- Removed the GraphQL API `csvFormattedRawEvents`.
```

because `tsvFormattedRawEvents` was not included in the previous release.

Also, I believe this line needs to be removed.
```
- Rename the `csvFormattedRawEvents` GraphQL API to `tsvFormattedRawEvents`.
```

Reply:
done